src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -701,8 +701,11 @@ private void CheckNewModifier(Symbol symbol, bool isNew, DiagnosticBag diagnosti
             }
         }
 
-        private static void CheckOverrideMember(Symbol overridingMember, OverriddenOrHiddenMembersResult overriddenOrHiddenMembers,
-            DiagnosticBag diagnostics, out bool suppressAccessors)
+        private void CheckOverrideMember(
```

Comment:
Should this PR target the `features/records` branch?
FYI, I am doing a refactoring of this method (simple breaking out of this massive and deeply nested method) in the records branch [https://github.com/dotnet/roslyn/pull/43275/commits/68426e7182b638d538ea16940a290dfc01e7bf9b](https://github.com/dotnet/roslyn/pull/43275/commits/68426e7182b638d538ea16940a290dfc01e7bf9b)
If covariant returns needs a dedicated feature branch, then consider cherry-picking that refactoring.

Reply:
Covariant returns are a separate feature.

---
In reply to: [413984024](https://github.com/dotnet/roslyn/pull/43576#discussion_r413984024) [](ancestors = 413984024)

Reply:
From standup meeting today, seems like folks are leaning towards using a separate branch for now.

Reply:
Your delta is in a private branch and not yet integrated. I don't see an advantage of merging now and hoping you don't change things further.  I know I will be changing things further.  Will deal with the conflicts later.

---
In reply to: [414044555](https://github.com/dotnet/roslyn/pull/43576#discussion_r414044555) [](ancestors = 414044555,413984024)